usage/ui.md

```diff
@@ -853,3 +853,114 @@ If you need to add, replace or delete an icon:
 ## Tailwind Configuration
 
 There are a few tailwind configuration additions on which the components rely (e.g. colors and additional shadows) and are therefore expected to use the tailwind config in this repository as basis (you can import it and extend it further if needed).
+
+## Dark Color Theme
+
+Dark color theme is more and more used on website today and many operating systems feature this functionality.
+Users might indicate their preference through the operating system setting or by interacting with a theme switcher component.
+
+Since we use Tailwind css with `class` strategy to manage dark mode. 
+This strategy had a down-side of not be able to manage the operating system preference.
+To by-pass this problem, we can use vanilla javascript and controlling both strategies.
+
+### Script
+
+```html
+<x-ark-dark-theme-script />
+```
+
+You can disable the script by adding `DARK_MODE_ENABLE=false` on your `.env` file.
+
+#### How to integrate
+The script should be inserted on each page in the `head` section. In our case, placing the script in the `app.blade.php` can do the trick.
+
+```html
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
+        ...
+
+        <title>...</title>
+        
+        <!-- place the script right after <title> to avoid FOUC -->
```

Comment:
Trivial but I had to google what FOUC meant so probably worth not "acronymizing" it :thinking: